sklearn/tests/test_calibration.py

```diff
@@ -182,6 +182,37 @@ def test_sample_weight(data, method, ensemble):
     assert diff > 0.1
 
 
+@pytest.mark.parametrize("method", ["sigmoid", "isotonic"])
+@pytest.mark.parametrize("ensemble", [True, False])
+def test_class_weight(data, method, ensemble):
+    n_samples = 100
+    n_classes = 2
+    class_weight = np.random.RandomState(seed=42).uniform(size=n_classes)
+    X, y = data
+    X_train, y_train = X[:n_samples], y[:n_samples]
+    X_test = X[n_samples:]
+
+    cw = dict(zip(np.arange(n_classes), class_weight))
+
+    base_estimator = LinearSVC(random_state=42)
+    calibrated_clf = CalibratedClassifierCV(
+        base_estimator, method=method, ensemble=ensemble, class_weight=cw
+    )
+    calibrated_clf.fit(X_train, y_train)
```

Comment:
The CI fails because of code coverage which reports that the following lines:
 - sklearn/calibration.py#L306
 - sklearn/calibration.py#L309-L310

are not covered. This happens because the case where `cv == "prefit"` for `CalibratedClassifierCV` is not tested.

Adding a test or parametrize this one on `cv` will cover those lines.

Reply:
Parametrized `prefit` in `test_class_weight.py`, thanks for your suggestion